source/security.rst

```diff
@@ -0,0 +1,38 @@
+.. _security:
+
+Security
+=================
+
+Introduction
+------------
+
+This document provides an overview of the security framework implemented in Open OnDemand, detailing security features, general policies, and procedures.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Security Topics
+
+   security/vulnerability-management
+   authentication/overview 
+   how-tos/monitoring/logging
+   customizations
```

Comment:
It's not clear to me why these are here. They seem to fill up the page, but it's not immediately clear how, say customizations, are related to security?

Reply:
> Logging and Auth are paramount in security, leave them.

Authentication is required either way - I can setup basic auth and call it a day, but that doesn't mean it's secure. And just having it in a ToC doesn't tell me that.

Counterintuitively, the more text you have on a page, the less any of it is read.

Reply:
I think at least the below are related to security. Maybe I can update the list and fold the whole "Security Topics" somehow so the links aren't in the way.

Announcements
Message of the Day (MOTD)
Overriding Pages
Add URLs to Help Menu
Configuration Profiles
Changing the Navigation bar
Interactive Apps Menu
Set Upload Limits
Set Download Limits
Block or Allow Directory Access
Disabling Users
Set Default SSH Host
Set SSH Allowlist
Set OOD SSH Port
Shell App SSH Command Wrapper
Fix Unauthorized WebSocket Connection in Shell App
Job Composer Script Size Limit
Hiding Job Arrays
Custom Error Page for Missing Home Directory on Launch
Customize Text in OnDemand
Disk Quota Warnings on Dashboard
Balance Warnings on Dashboard
Maintenance Mode
Disable Host Link in Batch Connect Session Card
Set Illegal Job Name Characters
XDMoD Integration
Accessing Remote File Systems
Cancel Interactive Sessions
Custom Pages
Support Tickets